templates/typescript_gapic/test/gapic-$service-$version.ts.njk

```diff
@@ -8,11 +8,16 @@ const {PassThrough} = require('stream');
 {% endif %}
 
 const FAKE_STATUS_CODE = 1;
-const error = new Error();
-error.code = FAKE_STATUS_CODE;
+class fakeError{
```

Comment:
class name should start with capital letter (`gts check` will yell on this line)

Reply:
You are right, thanks!